enhancements/security/runtime-class-scc.md

```diff
@@ -105,31 +106,33 @@ However, if admins happened to have added them, they will also have to allow the
 #### Point Values:
 Openshift SCCs need to be ordered by relative point value of "how restrictive" they are (a system described [here][byrestrictions].
 This section describes how points will be assigned for `RequiredRuntimeClasses`.
-In general, a smaller set of RequiredRuntimeClasses is more restricted. For instance, a list that is a strict subset of another is more restrictive, and thus should have
+In general, a smaller set of RequiredRuntimeClasses is more restricted. For instance, a list that is shorter than another is more restrictive, and thus should have
```

Comment:
with https://github.com/openshift/enhancements/pull/562#discussion_r559537004 I question this.

Reply:
or maybe make clearer what "restrictive" means. Correct: it matches less pods. Incorrect: when chosen it is more secure.